core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -316,6 +333,44 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Adds a {@link ChannelOption} of {@link Server}'s server sockets.
```

Comment:
How about `Sets the {@link ChannelOption} of the server socket bound by {@link Server}.`?